soc/arm/nuvoton_npcx/npcx7/Kconfig.defconfig.series

```diff
@@ -54,6 +54,11 @@ config SPI_NPCX_FIU
 	default y
 	depends on SPI
 
+config PECI_NPCX
+	default y
```

Comment:
should be able to drop the `default y` and `depends on PECI`

Reply:
This can be removed after adding your suggestion in drivers/peci/Kconfig.npcx] above :
```
default y
depends on DT_HAS_NUVOTON_NCPX_PECI_ENABLED
```